app/dashboard/helpers.py

```diff
@@ -325,7 +345,7 @@ def process_bounty_details(bountydetails, url, contract_address, network):
             expires_date=timezone.make_aware(timezone.datetime.fromtimestamp(bounty.get('deadline')), timezone=UTC),
             standard_bounties_id=bountyId,
             balance=bounty.get('balance'),
-            num_fulfillments=fulfillments.get('total', 0),
+            num_fulfillments=len(fulfillments),
```

Comment:
This will fail if `fulfillments` is None.  What happens if the bounty doesn't have any fulfillments yet?

Reply:
`fulfillments` will never be `None` because of the way fulfillments are constructed in `dashboard/utils#L91`.  they will also be class `list`.